src/content/docs/ai-observability/advanced-customization/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,306 @@
+---
+title: 'Remove sensitive data with drop filters'
+metaDescription: 'Drop rules prompts AI monitoring to drop attributes containing sensitive data.'
+freshnessValidatedDate: never
+---
+
+import aiDropFilterModal from 'images/ai_screenshot-crop_drop-filter-modal.webp'
+
+Our agents collect and store your data in the New Relic database (NRDB). When an agent captures an interaction with your AI assistant, it stores the contents of those prompts and responses as text string attributes within data events. This gives you greater insight into the cost, accuracy, and performance of your AI-powered app, but you may not want to store interactions that contain sensitive data.
```

Comment:
I'd add something to the end here that kind of bridges that you're talking about drop rules here. Something at the end like "This is where drop rules come in." (obviously not that one but you get the idea.